libraries/libopaevfio/opaevfiotest.c

```diff
@@ -47,7 +47,8 @@ void print_dfhs(struct opae_vfio *v)
 	struct fme_dfh *mmio = NULL;
 	uint8_t *p = NULL;
 
-	opae_vfio_region_get(v, 0, (uint8_t **)&mmio, NULL);
+	if (opae_vfio_region_get(v, 0, (uint8_t **)&mmio, NULL))
+		return;
```

Comment:
Better to print something when returning early.  As it is now, this will be a silent failure.